src/olympia/accounts/tests/test_utils.py

```diff
@@ -11,23 +12,18 @@
 
 
 FXA_CONFIG = {
-    'default': {
-        'client_id': 'foo',
-        'client_secret': 'bar',
-    },
     'other': {'client_id': 'foo_other', 'client_secret': 'bar_other'},
 }
```

Comment:
I don't think you should make this change - it leaves the `FXA_CONFIG` constant broken because it no longer has a `default` key, and `FXA_CONFIG` has to have a default as a fallback.  If you don't want to re-define `default` here you could do something like
`FXA_CONFIG = {**settings.FXA_CONFIG, 'other': {...}}` wdyt?

Or alternatively, if `FXA_CONFIG` isn't needed anywhere as a valid config for FxA, just define the `other` dict separately as  a constant.

Reply:
Looking a bit closer, it's kind of a moot point. The FXA_CONFIG variable in question is passed explicitly to the `config` argument for the `redirect_for_login` utility method. This function will use this config if explicitly passed, otherwise pulling the config from the request..

funny part is, everywhere in the code base we actually call the `redirect_for_login` function we don't pass the config so we have t his argument, and a test for it... kind of for no reason. I'll just remove it as probably something that was at one point used but not now.